source/_docs/guides/drupal8-commandline.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use the Command Line to complete Drupal Site Building Tasks with Using Terminus and Drush
+description: Learn how to add modules, and manage configuration between Pantheon environments.
+tags: [cli]
+categories: [develop, cli]
+type: guide
+permalink: docs/guides/:basename/
+contributors:
+  - stevector
+date: 2/15/2017
+---
+
+This guide will walk through using the command line to perform operations creating a new Drupal 8 site.
+With that site, we will add modules, create content, and move configuration between Pantheon environments.
+The tasks covered in this guide can all be done in the browser, without using the command line.
+For instance, seeing a list of all of your sites on Pantheon is easy in the browser.
+It is the first thing you see when signing into the Dashboard.
+
+![A list of websites in the Pantheon dashboard](/source/docs/assets/images/drupal8-commandline--dashboard-sites.png)
+
+You can see the same list on the command line by running:
+
+`terminus site:list`
+
+In essence, Terminus is a way to do on the command line everything you can do in Pantheon's browser-based dashboard.
+Once you are comfortable with Terminus, you may find it faster to use than the browser.
+Terminus also opens the door to writing scripts that combine multiple Terminus commands to accommodate whatever workflow needs you have.
+For more information about Terminus itself, see our [Terminus Manual](/docs/terminus/).
+
+## Installing and authenticating with Terminus
+
+The most reliable way to install Terminus is with our dedicated installer:
+
+`curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install`
+
+Once Terminus is installed, you can sign in with a [machine token generated in the Dashboard](https://dashboard.pantheon.io/machine-token/create).
+
+`terminus auth:login --machine-token=‹machine-token›`
+
+If you have trouble with these steps, [see our full installation documentation](https://pantheon.io/docs/terminus/install/).
+
+## Create your site with Dev, Test, and Live environments
+
+#### See your options for creating new sites
+
+Pantheon offers a few choices for spinning up new sites.
+You can see them all with.
+
+`terminus upstream:list`
+
+The one we want is `Drupal 8`.
+
+`terminus upstream:list | grep "Drupal 8"`
+
+That command shows us the unique ID for our Drupal 8 source repository.
+We will copy that ID (8a129104-9d37-4082-aaf8-e6f31154644e) into the next command for creating a new site.
```

Comment:
Cool, I'll also remove the part above about grepping the output of `terminus upstream:list`